odroid-xu3-fan-control.service

```diff
@@ -0,0 +1,19 @@
+# This file was autogenerated by node-deb 0.1.6
+
+[Unit]
+Description=odroid-xu3-fan-control
+
+[Service]
+Type=simple
+WorkingDirectory=/home/odroid/odroid-xu3-fan-control
+ExecStart=/usr/home/odroid/odroid-xu3-fan-control/odroid-xu3-fan-control.sh
```

Comment:
I've just tested this. But doesn't work. Is `/usr/home/..` prefix an issue? Typo?

    nthx@odroid:~$ sudo cp odroid-xu3-fan-control.service /etc/systemd/system

    nthx@odroid:~$ ll /home/odroid/odroid-xu3-fan-control
    total 16
    -rw-rw-r-- 1 nthx nthx 1080 Nov 23 23:34 LICENSE
    -rwxrwxr-x 1 nthx nthx 1963 Nov 23 23:34 odroid-fan-controller*
    -rwxrwxr-x 1 nthx nthx 2591 Nov 23 23:34 odroid-xu3-fan-control.sh*
    -rw-rw-r-- 1 nthx nthx 1431 Nov 23 23:38 README.md

    nthx@odroid:~$ sudo service odroid-fan-controller start
    Failed to start odroid-fan-controller.service: Unit odroid-fan-controller.service failed to load: No such file or   directory.